package.json

```diff
@@ -19,6 +19,13 @@
     "start:playground": "npm run build && node test/app.js",
     "start": "npm run docs && npx gulp default",
     "std-version": "standard-version -m \"chore(release): version %s build ${TRAVIS_BUILD_NUMBER} [ci skip]\"",
+    "style:assets": "./scripts/copy-assets.sh",
+    "style:compile": "node-sass -q --output-style expanded --precision 5 scss/ --output dist/",
+    "style:less": "npx gulp pkg-less",
+    "style:minify": "foreach -t -g \"dist/*.css\" -x \"cleancss -O1 specialComments:1 #{path} -o #{dir}/#{name}.min.css\" --no-c && foreach -t -g \"dist/components/*.css\" -x \"cleancss -O1 specialComments:1 #{path} -o #{dir}/#{name}.min.css\" --no-c",
+    "style:postCSS": "postcss --config config/postcss.config.js  --replace dist/**/*.css && postcss --config config/postcss.config.js  --replace dist/*.css",
+    "style:remove": "if [ -d dist/ ]; then rm -rf dist/; fi",
+    "style:rename": "mv dist/all.css dist/fiori-fundamentals.css && mv dist/all-ie11.css dist/fiori-fundamentals-ie11.css && rm dist/fundamentals-ie11.less.css dist/fundamentals.less.css",
```

Comment:
These would definitely break if run out of order. This is the path that the bootstrap repo takes, but I'm totally open to other opinions! https://github.com/twbs/bootstrap/blob/master/package.json#L24

Reply:
Also my overarching plan is to get rid of `style:rename`, `style:less` and `style:assets` in the future.

Reply:
I think these are okay to leave for now since this PR is about achieving parity.  It might be a nice follow-up task to optimize all this once it's all done.